kubelet/datadog_checks/kubelet/common.py

```diff
@@ -21,6 +21,13 @@ def is_excluded(name, image):
 
 CADVISOR_DEFAULT_PORT = 0
 
+def urljoin(*args):
+    """
+    Joins given arguments into an url. Trailing but not leading slashes are
+    stripped for each argument.
+    :return: string
+    """
+    return return '/'.join(map(lambda x: x.strip('/'), args))
```

Comment:
As you need a change, may I suggest:

`return '/'.join(arg.strip('/') for arg in args)` ?

Reply:
Although we are dealing with 2 args, I guess your approach is more pythonic and will avoid creating the map. I'll update, thanks for the suggestion!